openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -339,10 +338,10 @@ public void execute(StfCoreExtension test) throws Exception {
 					addModules = "--add-modules java.xml.ws,java.corba";
 				}
 				javatestAgent = test.doRunBackgroundProcess("Starting javatest agent", "AGNT", ECHO_ON, ExpectedOutcome.neverCompletes(), test.createJavaProcessDefinition()
-						.addJvmOption("-Djavatest.security.allowPropertiesAccess=true -Djava.security.policy=" + test.env().findPrereqFile(jckTopDir + jckVersion + "/" + testSuiteFolder + "/lib/jck.policy").toString())
+						.addJvmOption("-Djavatest.security.allowPropertiesAccess=true -Djava.security.policy=" + test.env().findPrereqFile(testSuiteFolder + "/lib/jck.policy").toString())
```

Comment:
Should we include `jckRoot`? Without it, is it possible to find the file in the wrong jck version? Same for the rest of `findPrereqFile()`

Reply:
When JckRoot is provided as argument to Jck.java, which is the case for us here, jckRoot == jckTopDir.

Reply:
Thanks for clarifying, Mesbah. :)